patch/llpcPatchDescriptorLoad.cpp

```diff
@@ -633,7 +633,10 @@ Value* PatchDescriptorLoad::LoadDescriptor(
             auto pCastedDescPtr = CastInst::Create(Instruction::BitCast, pDescPtr, pDescPtrTy, "", pInsertPoint);
 
             // Load descriptor
-            pCastedDescPtr->setMetadata(m_pContext->MetaIdUniform(), m_pContext->GetEmptyMetadataNode());
+            MDNode *MD = m_pContext->GetEmptyMetadataNode();
```

Comment:
I think we can remove this. And use m_pContext->GetEmptyMetadataNode() directly in the function calls.

Reply:
Doesn't GetEmptyMetadataNode create an empty MDNode?  Wouldn't this just create 2 MDNodes, one with uniform and one with invariant.load? Is this somehow faster? I think calling the same function twice also makes this less readable but that is subjective. I don't have an issue doing it though, eh.

Reply:
The function will not create the metadata node twice. The metadata is created when LLPC context is initialized and the function just returns it whenever we want to use it.